ReportPortal.SpecFlowPlugin/EventArguments/StepFinishedEventArgs.cs

```diff
@@ -14,12 +15,26 @@ public StepFinishedEventArgs(Service service, AddLogItemRequest request, TestRep
             TestReporter = testReporter;
         }
 
+        public StepFinishedEventArgs(Service service, AddLogItemRequest request, TestReporter testReporter, FeatureContext featureContext, ScenarioContext scenarioContext, ScenarioStepContext stepContext)
+            : this(service, request, testReporter)
+        {
+            FeatureContext = featureContext;
```

Comment:
Do we really have to keep specflow context? I think user can get it via `FeatureContext.Current` at any place. Not sure, need verify it. I remember that it doean't work in case of parallel execution. PLease approve.

Reply:
Nikolay and I discussed it offline and came to agreement to keep proposed changes due to difficulty to get access to SpecFlow contexts from static event handlers.